src/coreclr/tools/aot/ILCompiler.Compiler/ILCompiler.Compiler.csproj

```diff
@@ -448,6 +448,12 @@
     <Compile Include="..\..\Common\JitInterface\CorInfoInstructionSet.cs">
       <Link>JitInterface\CorInfoInstructionSet.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\JitInterface\CorInfoHelpFunc.cs">
+      <Link>JitInterface\CorInfoHelpFunc.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\JitInterface\CorInfoTypes.cs">
```

Comment:
Sight, I thought we already included this file when I wrote https://github.com/dotnet/runtime/pull/53567#discussion_r648939767.